File: reset_hash.js
const { PrismaClient } = require('@prisma/client');
const path = require('path');

process.env.DATABASE_URL = 'file:C:/Users/gkjuw/AppData/Roaming/ogenti/data/ogenti.db';

const prisma = new PrismaClient();

async function main() {
  const result = await prisma.agentProfile.updateMany({
    where: { NOT: { selfPromptHash: '' } },
    data: { selfPromptHash: '' },
  });
  console.log('Reset:', result.count, 'agents');
}

main().then(() => prisma.$disconnect()).catch(e => { console.error(e); prisma.$disconnect(); process.exit(1); });

File: reset_hash.py
import sqlite3
db = sqlite3.connect(r'C:\Users\gkjuw\AppData\Roaming\ogenti\data\ogenti.db')
cur = db.cursor()
cur.execute("UPDATE AgentProfile SET selfPromptHash = '' WHERE selfPromptHash IS NOT NULL AND selfPromptHash != ''")
print(f'Reset selfPromptHash for {cur.rowcount} agents')
db.commit()
db.close()
